license-compliance/gradle-license-report/license-normalizer-bundle.json

```diff
@@ -1,156 +1,371 @@
+    },
+    {
+      "bundleName": "zlib-mit",
+      "licenseName": "MIT OR Zlib",
+      "licenseUrl": "https://spdx.org/licenses/MIT.html OR https://spdx.org/licenses/Zlib.html"
+    },
+    {
+      "bundleName": "lpgl20-mit",
+      "licenseName": "LGPL-2.0 OR MIT",
+      "licenseUrl": "https://spdx.org/licenses/LGPL-2.0-only.html OR https://spdx.org/licenses/MIT.html"
+    },
+    {
+      "bundleName": "cc0-1-mit",
+      "licenseName": "MIT OR CC0-1.0",
+      "licenseUrl": "https://spdx.org/licenses/MIT.html OR https://spdx.org/licenses/CC0-1.0.html"
+    },
+    {
+      "bundleName": "mit-0",
+      "licenseName": "MIT-0",
+      "licenseUrl": "https://spdx.org/licenses/MIT-0.html"
+    },
+    {
+      "bundleName": "lgpl2.1-or-later",
+      "licenseName": "LGPL-2.1+",
+      "licenseUrl": "https://spdx.org/licenses/LGPL-2.1+.html"
+    },
+    {
+      "bundleName": "apache2",
+      "licenseName": "Apache-2.0",
+      "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0.txt"
+    },
+    {
+      "bundleName": "bsd2clause",
+      "licenseName": "BSD-2-Clause",
+      "licenseUrl": "https://opensource.org/licenses/BSD-2-Clause"
+    },
+    {
+      "bundleName": "bsd3clause",
+      "licenseName": "BSD-3-Clause",
+      "licenseUrl": "https://opensource.org/licenses/BSD-3-Clause"
+    },
+    {
+      "bundleName": "cc0-1",
+      "licenseName": "CC0-1.0",
+      "licenseUrl": "http://creativecommons.org/publicdomain/zero/1.0/"
+    },
+    {
+      "bundleName": "cddl1",
+      "licenseName": "CDDL-1.0",
+      "licenseUrl": "https://opensource.org/licenses/CDDL-1.0"
+    },
+    {
+      "bundleName": "epl2",
+      "licenseName": "EPL-2.0",
+      "licenseUrl": "https://www.eclipse.org/legal/epl-2.0/"
+    },
+    {
+      "bundleName": "gpl2.0",
+      "licenseName": "GPL-2.0",
+      "licenseUrl": "https://spdx.org/licenses/GPL-2.0.html"
+    },
+    {
+      "bundleName": "lgpl2.1",
+      "licenseName": "LGPL-2.1",
+      "licenseUrl": "https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html"
+    },
+    {
+      "bundleName": "mit",
+      "licenseName": "MIT",
+      "licenseUrl": "http://www.opensource.org/licenses/mit-license.php"
+    },
+    {
+      "bundleName": "mpl1.1",
+      "licenseName": "MPL-1.1",
+      "licenseUrl": "https://spdx.org/licenses/MPL-1.1.html"
+    },
+    {
+      "bundleName": "upl1.0",
+      "licenseName": "UPL-1.0",
+      "licenseUrl": "https://spdx.org/licenses/UPL-1.0.html"
+    }
+  ],
+  "transformationRules": [
+    {
+      "bundleName": "apache2",
+      "licenseNamePattern": "^Apache 2$"
+    },
+    {
+      "bundleName": "apache2",
+      "licenseNamePattern": "^\"Apache[-\\s]2\\.0\";link=\"https?://www.apache.org/licenses/LICENSE-2.0\\.txt\"$"
+    },
+    {
+      "bundleName": "apache2",
+      "licenseNamePattern": "^Apache 2\\.0$"
+    },
+    {
+      "bundleName": "apache2",
+      "licenseNamePattern": "^Apache-2\\.0$"
+    },
```

Comment:
```suggestion
    },
    {
      "bundleName": "apache2",
      "licenseNamePattern": "^\"Apache-2\\.0\";link=\"https://www.apache.org/licenses/LICENSE-2\\.0\\.txt\"$"
    },
```

Reply:
we have the following pattern for the line like this
https://github.com/th2-net/.github/blob/662510364aee02fc85e36ced07448be8412dae53/license-compliance/gradle-license-report/license-normalizer-bundle.json#L156

Do you think it is not enough ?

Reply:
Probably @Nikita-Smirnov-Exactpro should clarify what he meant by the suggestion.

Reply:
This comment is result of transformationRules block compaction in different files.
@Andrei-Shulika could you please change existed pattern from `...LICENSE-2.0...` to `LICENSE-2\\.0` because pattern is regular expression.

Reply:
done, check please